src/lilURL.php

```diff
@@ -82,27 +83,33 @@ public function handlePOST($id = null, $user = null)
                 'utm_source' => $_POST['gaSource'],
                 'utm_medium' => $_POST['gaMedium'],
                 'utm_term' => $_POST['gaTerm'],
-                'utm_content' =>$_POST['gaContent'],
+                'utm_content' => $_POST['gaContent'],
                 'utm_campaign' => $_POST['gaName'],
             ]);
 
-            $longurl .=  (strpos($_POST['theURL'], '?') !== false) ? '&' : '?';
+            $longurl .= (strpos($_POST['theURL'], '?') !== false) ? '&' : '?';
             $longurl .= $gaTags;
         }
+        // Check to see if the pair already exists in db
```

Comment:
It's more important to check to see if the alias is used before checking if the pair are used.